taskcluster/kinds/finetune-student/kind.yml

```diff
@@ -47,7 +48,8 @@ tasks:
             cache:
                 type: finetune-student
                 resources:
-                    - pipeline/train/configs/model/student.yml
+                    - pipeline/train/configs/model/student.tiny.yml
+                    - pipeline/train/configs/model/student.base.yml
```

Comment:
If you'd like to avoid rebuilding `tiny` tasks when only `base` changes you can avoid hardcoding both of these and using something like:
```
resources:
    - pipeline/train/configs/model/student.{student_model}.yml
```

And adding this to `task-context`:
```
task-context:
    from-parameters:
        student_model: training_config.experiment.student-model
```

...which should end up with just the actively used `student-model` yml file in the cache digest.

I know we're largely relying on `previous_group_ids` and `existing_tasks` rather than cached tasks, but this might still be useful to do.